inline.go

```diff
@@ -75,6 +76,9 @@ func (p *parser) inline(data []byte) {
 			if p.inlineCallback[data[end]] != nil {
 				if end+1 < len(data) && data[end] == 'h' && data[end+1] != 't' {
 					end++
+				} else if strings.ContainsRune("hHmMfF", rune(data[end])) && !isAutoLink(p, data, end) {
```

Comment:
ContainsRune returns a boolean, IndexByte returns the position. While you can achieve the same result with both, I believe it the be more idiomatic to use the function that bears the name of what you want to do,